ci-operator/step-registry/idp/htpasswd/idp-htpasswd-commands.sh

```diff
@@ -3,6 +3,49 @@
 set -e
 set -u
 set -o pipefail
+set -x
+
+function check_if_hypershift_env () {
+    if [ -f "${SHARED_DIR}/nested_kubeconfig" ]; then
+        IS_HYPERSHIFT_ENV="yes"
+    else
+        IS_HYPERSHIFT_ENV="no"
```

Comment:
Yes it is needed, because of the existing `set -u`, which will make `if [ "$IS_HYPERSHIFT_ENV" == "yes" ]; then` fail in OCP CI where the var has not been set.